internal/command/image/update.go

```diff
@@ -49,12 +54,36 @@ The update will perform a rolling restart against each VM, which may result in a
 			Name:        "detach",
 			Description: "Return immediately instead of monitoring update progress",
 		},
+		flag.Bool{
+			Name:        "auto-confirm",
+			Description: "Will automatically confirm changes without an interactive prompt.",
+		},
 	)
 
 	return cmd
 }
 
-func runUpdate(ctx context.Context) error {
+func runUpdate(ctx context.Context) (err error) {
+	var (
+		appName = app.NameFromContext(ctx)
+		client  = client.FromContext(ctx).API()
+	)
+
+	app, err := client.GetAppCompact(ctx, appName)
+	if err != nil {
+		return fmt.Errorf("get app: %w", err)
+	}
+
```

Comment:
Looks like we could save a graphql query if we just pass app into updateImageForNomad and updateImageForMachines